config/routes.rb

```diff
@@ -110,6 +110,8 @@
 
     resources :cohorts, only: %i[index show]
 
+    resources :patients, only: %i[index], module: "programme"
```

Comment:
Nice use of the module, I've been wondering about whether we should structure more controllers like this as we've got a few different views of the same data in different sections of the app.